src/primitives/transaction.h

```diff
@@ -322,6 +327,10 @@ class CTransaction
         READWRITE(*const_cast<uint32_t*>(&nLockTime));
         if (nVersion >= 2) {
             READWRITE(*const_cast<std::vector<CPourTx>*>(&vpour));
+            if (vpour.size() > 0) {
+                READWRITE(*const_cast<CPubKey*>(&joinSplitPubKey));
+                READWRITE(*const_cast<std::vector<unsigned char>*>(&joinSplitSig));
+            }
```

Comment:
I guess this is a general problem with the Bitcoin serialization API: fields are declared as ``const``, but ``READWRITE`` can write to them. Is modifying a const variable even defined behaviour?